### PR TITLE
CS_PT34_PT35

### PR DESCRIPTION
**Pull request by csonparote**
```
@PF_CreateAccount_ResendCode
Scenario: Resend verification code
Given User is on home page
When User begins account creation
And creates new credentials
And User clicks resend confirmation code
And enters confirmation code
Then Validate that new code is accepted

@PF_CreateAccount_InvalidCode
Scenario: Resend verification code
Given User is on home page
When User begins account creation
And creates new credentials
And User enters invalid code
Then Validate that error is displayed for invalid code

```

### CODE REVIEW

File: platform/src/test/java/platformstepdefinition/PF_CreateAccount.java [hunk 1]
```diff
@@ -90,4 +92,50 @@ public void enters_confirmation_code() throws Throwable {
...
+		Assert.assertTrue(CommonFunctions.isElementVisible(createAccountStep3.passwordField));
+
+	}
+
+	@And("^User clicks resend confirmation code$")
+	public void user_clicks_resend_confirmation_code() throws Throwable {
+		// throw new PendingException();
+		CreateAccountStep2 createAccountStep2 = new CreateAccountStep2(driver);
+		createAccountStep2.clickResendCodeLink();
+		NewBaseClass.validateError(null, "確認コードを再送しました、宛先メールをご確認ください。");
```
Comment (Sopan-Patil) [inline platform/src/test/java/platformstepdefinition/PF_CreateAccount.java]: Please do not use the content. You can use it through excel.

File: platform/src/test/java/platformstepdefinition/PF_CreateAccount.java [hunk 2]
```diff
@@ -90,4 +92,50 @@ public void enters_confirmation_code() throws Throwable {
...
+	 * @And("^enters new confirmation code$") public void
+	 * enters_new_confirmation_code() throws Throwable { // throw new
+	 * PendingException(); CreateAccountStep2 createAccountStep2 = new
+	 * CreateAccountStep2(driver); createAccountStep2.addConfirmationCode(); }
+	 */
+
+	@Then("^Validate that error is displayed for invalid code$")
+	public void validate_that_error_is_displayed_for_invalid_code() throws Throwable {
+		// throw new PendingException();
+		NewBaseClass.validateError(null, "6桁の確認コードを入力してください。");
```
Comment (Sopan-Patil) [inline platform/src/test/java/platformstepdefinition/PF_CreateAccount.java]: Please do not use the content. You can use it through excel.

File: platform/src/test/java/platformstepdefinition/PF_CreateAccount.java [hunk 3]
```diff
@@ -90,4 +92,50 @@ public void enters_confirmation_code() throws Throwable {
...
+		// throw new PendingException();
+		CreateAccountStep2 createAccountStep2 = new CreateAccountStep2(driver);
+		createAccountStep2.clickResendCodeLink();
+		NewBaseClass.validateError(null, "確認コードを再送しました、宛先メールをご確認ください。");
+
+	}
+
+	/*
+	 * @And("^enters new confirmation code$") public void
+	 * enters_new_confirmation_code() throws Throwable { // throw new
```
Comment (Sopan-Patil) [inline platform/src/test/java/platformstepdefinition/PF_CreateAccount.java]: Commented code pl remove

**Review by Sopan-Patil — CHANGES_REQUESTED** [review]

File: platform/src/test/java/platformstepdefinition/PF_CreateAccount.java (hunk 1 above)
Comment (csonparote) [reply platform/src/test/java/platformstepdefinition/PF_CreateAccount.java]: i am validating error message here. Cud not find error string in excel

File: platform/src/test/java/platformstepdefinition/PF_CreateAccount.java (hunk 2 above)
Comment (csonparote) [reply platform/src/test/java/platformstepdefinition/PF_CreateAccount.java]: I am validating error message here. Cud not find error string in excel

File: platform/src/test/java/platformstepdefinition/PF_CreateAccount.java (hunk 3 above)
Comment (csonparote) [reply platform/src/test/java/platformstepdefinition/PF_CreateAccount.java]: done

File: platform/src/test/java/platformstepdefinition/PF_CreateAccount.java (hunk 1 above)
Comment (csonparote) [reply platform/src/test/java/platformstepdefinition/PF_CreateAccount.java]: done

File: platform/src/test/java/platformstepdefinition/PF_CreateAccount.java (hunk 2 above)
Comment (csonparote) [reply platform/src/test/java/platformstepdefinition/PF_CreateAccount.java]: done

File: platform/src/test/java/platformstepdefinition/PF_CreateAccount.java (hunk 3 above)
Comment (csonparote) [reply platform/src/test/java/platformstepdefinition/PF_CreateAccount.java]: done

**Comment by csonparote** [comment]
Changes done.
Please review changed files

**Comment by Sopan-Patil** [comment]
Please resolved conflicts 
Common/Common/src/main/java/utils/XLHandler.java
platform/src/main/java/base/NewBaseClass.java
platform/src/test/java/platformfeatures/PF_CreateAccount.feature
platform/src/test/java/platformrunner/Runner.java
platform/src/test/java/platformstepdefinition/PF_CreateAccount.java

File: platform/src/test/java/platformstepdefinition/PF_CreateAccount.java [hunk 4]
```diff
@@ -109,6 +110,51 @@ public void enters_confirmation_code() throws Throwable {
...
+		NewBaseClass newBaseClass = new NewBaseClass();
+		newBaseClass.validateMessageFromExcel("InvalidCodeError", "//p[@class='alert__des']");
+
+	}
+
+	@And("^User enters invalid code$")
+	public void user_enters_invalid_code() throws Throwable {
+		// throw new PendingException();
+		CreateAccountStep2 createAccountStep2 = new CreateAccountStep2(driver);
+		// createAccountStep2.convertOTPToList();
```
Comment (Sopan-Patil) [inline platform/src/test/java/platformstepdefinition/PF_CreateAccount.java:149]: remove commented code

**Review by Sopan-Patil — APPROVED** [review]

**Pull request merged**